core_types/src/entry.rs

```diff
@@ -9,7 +12,25 @@ use std::ops::Deref;
 /// Structure holding actual data in a source chain "Item"
 /// data is stored as a JSON string
 #[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct Entry(String);
+pub struct Entry {
+    value: Content,
```

Comment:
I think its because it will implement AddressableContent trait which has `content()`, so we can't have a private field named content otherwise you would have to break convention for getter name.

Reply:
@lucksus @ddd-mtl yes exactly.